datamodel/upgrade_scripts/3.0.2-4.0.0/oracle/rsgb.sql

```diff
@@ -10,6 +10,11 @@ INSERT INTO aard_verkregen_recht (aand, omschr_aard_verkregenr_recht) VALUES ('2
 INSERT INTO aard_recht_verkort (aand, omschr) VALUES ('25', 'Huur afhankelijk opstal (recht van)');
 INSERT INTO aard_recht_verkort (aand, omschr) VALUES ('26', 'Pachtafhankelijk opstal (recht van)');
 
+--ihkv https://b3partners.atlassian.net/browse/BRMO-381 zouden onderstaande materialized views verwijderd kunnen worden. ivm potentiële afhankelijkheid wordt dit niet automatisch gedaan
```

Comment:
dit dient ook in de upgrade procedure te worden beschreven.

Reply:
Ik heb een stuk tekst toegevoegd onder: https://github.com/B3Partners/brmo/wiki/Upgrade-v3.0.2-naar-v4.0.0#upgrade-scripts-2